core/src/main/java/org/apache/gravitino/audit/AuditLogManager.java

```diff
@@ -0,0 +1,90 @@
+package org.apache.gravitino.audit;
+
+import com.google.common.annotations.VisibleForTesting;
+import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.gravitino.exceptions.GravitinoRuntimeException;
+import org.apache.gravitino.listener.EventListenerManager;
+import org.apache.gravitino.listener.api.EventListenerPlugin;
+import org.apache.gravitino.listener.api.event.Event;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AuditLogManager {
+
+  private static final Logger LOG = LoggerFactory.getLogger(AuditLogManager.class);
+
+  @VisibleForTesting private AuditLogWriter auditLogWriter;
+
+  public void init(Map<String, String> properties, EventListenerManager eventBusManager) {
+    LOG.info("Audit log properties {}", properties);
+    AuditLogConfig auditLogConfig = new AuditLogConfig(properties);
+    if (!auditLogConfig.isAuditEnabled()) {
+      LOG.warn("Audit log is not enabled");
+      return;
+    }
+
+    LOG.info("Audit log config {}", auditLogConfig);
+    String writerClassName = auditLogConfig.getWriterClassName();
+    String formatterClassName = auditLogConfig.getAuditLogFormatterClassName();
+    Formatter formatter = loadFormatter(formatterClassName);
+    LOG.info("Audit log writer class name {}", writerClassName);
+    if (StringUtils.isEmpty(writerClassName)) {
+      throw new GravitinoRuntimeException("Audit log writer class is not configured");
+    }
+
+    auditLogWriter =
+        loadAuditLogWriter(
+            writerClassName, auditLogConfig.getWriterProperties(properties), formatter);
+
+    eventBusManager.addEventListener(
```

Comment:
The event listener mode is `SYNC` by default which means will block the main process if writer blocks, How about using `ASYNC_ISOLATED` by default?